example_14/hardware/rrbot_actuator_without_feedback.cpp

```diff
@@ -218,18 +211,18 @@ hardware_interface::return_type ros2_control_demo_example_14::RRBotActuatorWitho
   const rclcpp::Time & /*time*/, const rclcpp::Duration & /*period*/)
 {
   // START: This part here is for exemplary purposes - Please do not copy to your production code
-  RCLCPP_INFO(
-    rclcpp::get_logger("RRBotActuatorWithoutFeedback"), "Writing command: %f", hw_joint_command_);
+  std::stringstream ss;
+  ss << "Writing..." << std::endl;
+  ss << std::fixed << std::setprecision(2) << std::endl;
```

Comment:
Otherwise we end up with an empty line between "writing..." and the command.
```suggestion
  ss << std::fixed << std::setprecision(2);
```